aspnetcore/release-notes/aspnetcore-9/includes/customize-named-pipes.md

```diff
@@ -0,0 +1,28 @@
+### Customize Kestrel named pipe endpoints
+
+Kestrel's named pipe support has been improved with advanced customization options. The new `CreateNamedPipeServerStream` method on the named pipe options allows pipes to be customized per-endpoint.
```

Comment:
@tdykstra, minor item:  is per-endpoint a property or should this be "per endpoint"

Reply:
```suggestion
Kestrel's named pipe support has been improved with advanced customization options. The new `CreateNamedPipeServerStream` method on the named pipe options allows pipes to be customized per endpoint.
```